### ISSUE TITLE
Puppeteer Won't Launch on Raspberry Pi

### ISSUE THREAD

**Issue by johnentrieri** [johnentrieri/Democratic-Meals#1]
Receive an _Error: Failed to launch chrome!_ when running the **Blue Apron Puller** off of a Raspberry Pi 4.

![image](https://user-images.githubusercontent.com/7882002/70108821-2ee4e380-1618-11ea-93bc-7019370f85ff.png)

**Comment by johnentrieri** [comment]
Found a workaround in the following comment:

https://github.com/puppeteer/puppeteer/issues/550#issuecomment-551991273

There seems to be many issues with running chromium in headless for the Raspberry Pi - the user who wrote the comment worked around this by creating an _X Virtual Frame Buffer (xvfb)_ (essentially a fake display) and aiming chromium (not in headless mode) at this display.

Ran `sudo apt-get install xvfb`

Started a buffer with `Xvfb :3 -screen 0 1920x1080x24 &`

Launched Puppeteer as follows:

```
const browser = await puppeteer.launch({
        executablePath: '/usr/bin/chromium-browser',
        headless: false,
        defaultViewport: { height: 1080, width: 1920 },
        args: [
            '--display=:3',
            '--no-sandbox',
            '--disable-extensions'
        ]
    });

```
Worked, but took a decently long time

**Comment by johnentrieri** [comment]
Fixed this issue in: 53892be450999f8339a7f18810233da628c4c632
Created a seperate `app.js.pi` file that contains the above workaround.

Also created a bash script `pi-pull` that does the following:
  1. Backs up standard app.js file, and copies app.js.pi as the main app.js file
  2. Runs an X Virtual Frame Buffer (Xvfb)
  3. Runs the Blue Apron Puller Script
  4. Kills the Xvfb
  5. Resets the app.js to the standard (Windows) version
